applications/machine_learning/src/modules/ble_bond.c

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
Consider moving this module to CAF. It has no additional dependencies and it could be useful for other applications that want to support Bluetooth settings (it would work as very simple, default implementation).
That could be done in a separate PR.

Reply:
Created a task for it.